web/src/app/alerts/AlertDetailLogs.tsx

```diff
@@ -102,7 +111,7 @@ export default function AlertDetailLogs(props) {
     )
   }
 
-  const getLogStatusClass = (status) => {
+  const getLogStatusClass = (status: string): string => {
```

Comment:
Can we change this to `NotificationStatus` and add a check that the switch is exhaustive?